native/effektio/api.rsh

```diff
@@ -94,6 +94,25 @@ object RoomMessage {
 
     /// the server receiving timestamp
     fn origin_server_ts() -> u64;
+
+    fn msgtype() -> string;
+
+    fn image_description() -> Future<Result<ImageDescription>>;
+}
+
+object ImageDescription {
+
+    fn bin_data() -> Future<Result<buffer<u8>>>;
```

Comment:
can we have a better name for this function? How about `image_binary` or something?

Reply:
Great idea